src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -21,7 +21,7 @@ function checkFeedbackQuestion(form) {
         setStatusMessage(DISPLAY_FEEDBACK_QUESTION_TEXTINVALID, StatusType.DANGER);
         return false;
     }
-    if ($(form).find('[name=' + FEEDBACK_QUESTION_TYPE + ']').val() === 'NUMSCALE') {
+    if ($(form).find('[id=' + FEEDBACK_QUESTION_TYPE + ']').val() === 'NUMSCALE') {
```

Comment:
The forms for adding and editing questions are different right? For editing old questions, does validating numscale question still work if you change this?